kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/user/LibraryFolderWatcher.java

```diff
@@ -43,10 +43,6 @@
 import java.nio.file.Files;
```

Comment:
Please update license header here in line 2 from `* Copyright (c) 2017, 2021 Gluon and/or its affiliates.` to `* Copyright (c) 2017, 2022 Gluon and/or its affiliates.`.